frontend/src/streamDetails/StreamDeviceDetailsCard.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Grid } from "@material-ui/core";
+
+import DeviceInfo from "../model/DeviceInfo";
+import DashboardCard from "../general/dashboard/DashboardCard";
+import SimpleTable from "../general/simpleTable/SimpleTable";
+
+export default function StreamDeviceDetailsCard(props) {
+  const { cardTitle, device, channel } = props;
+
+  const propertyNames = ["Name", "Serial Number", "Channel"];
+  const properties = [device.name, device.serialNumber, channel];
+
+  return (
+    <>
```

Comment:
You don't need to use the react.fragment since thw componenet is wrapped in a dashboard card

Reply:
8654c23 :rocket: